File: easy-solution/minesp3164/283.py
class Solution(object):
    def moveZeroes(self, nums):
        count = 0
        for i in range(len(nums)):
            if(nums[i] != 0):
                nums[count] = nums[i]
                count+= 1
        for i in range(count,len(nums)):
            nums.insert(i, 0)
            nums.pop()


        """
        :type nums: List[int]
        :rtype: None Do not return anything, modify nums in-place instead.
        """

File: easy-solution/Antidote21/283.c
void moveZeroes(int* nums, int numsSize){
    int i,j,length=0;
    for(i=0;i<numsSize;i++){
        if(nums[i]!=0){
            nums[length] = nums[i];
            length++;
        }      
    }


    for(j=length;j<numsSize;j++){
         nums[j] = 0;          
    }


    return nums;

} 
